src/helpers/zones.ts

```diff
@@ -65,6 +65,9 @@ export function isZoneValid(zone: Zone): boolean {
   if (isNaN(zone.bottom) || isNaN(zone.top) || isNaN(zone.left) || isNaN(zone.right)) {
     return false;
   }
+  if (zone.top < 0 || zone.left < 0 || zone.right < 0 || zone.bottom < 0) {
+    return false;
+  }
```

Comment:
You're 100% right, but out of curiosity what made you change that in this particular PR ? I do'nt think it breaks any test ATM. (Come to think of it, it really _should_ break a test, might be worth adding one)

Reply:
@hokolomopo Changing it can have the correct error text (from something like `undefined for rows` to `invalid reference`) in error tooltip. 
I think the reason why it didn't break the tests is because the error is still correctly caught by us, so it will still show in the error tooltip, and the app won't crash.